companies/services/kis_quote.py

```
@@ -0,0 +1,220 @@
+                logger.error(
+                    f"KIS 시세 조회 요청 실패 ({stock_code}): "
+                    f"{response.status_code} Server Error: {response.reason} for url: {response.url}"
+                )
+                # 응답 본문이 있으면 로깅
+                try:
+                    error_body = response.text[:500]  # 최대 500자만
+                    if error_body:
+                        logger.debug(
+                            f"KIS API 에러 응답 본문 ({stock_code}): {error_body}"
+                        )
+                except:
```

Ruff (0.14.11)

Do not use bare `except`

(E722)